l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -672,24 +672,27 @@ def setDatiRiepilogo(self, invoice, body):
     def setDatiPagamento(self, invoice, body):
         if invoice.payment_term:
             DatiPagamento = DatiPagamentoType()
-            if not invoice.payment_term.fatturapa_pt_id:
+
+            if not invoice.fatturapa_pt_id:
                 raise UserError(
                     _('Payment term %s does not have a linked e-invoice '
                       'payment term') % invoice.payment_term.name)
-            if not invoice.payment_term.fatturapa_pm_id:
+
+            if not invoice.fatturapa_pm_id:
                 raise UserError(
                     _('Payment term %s does not have a linked e-invoice '
                       'payment method') % invoice.payment_term.name)
+
             DatiPagamento.CondizioniPagamento = (
-                invoice.payment_term.fatturapa_pt_id.code)
+                invoice.fatturapa_pt_id.code)
```

Comment:
`fatturapa_pt_id` è introdotto da `l10n_it_fiscal_payment_mode` ma `l10n_it_fatturapa_out` non dipende da `l10n_it_fiscal_payment_mode`

Reply:
I campi invoice.fatturapa_pt_id e invoice.fatturapa_pm_id sono introdotti in l10n_it_fiscal_payment_term con questa PR (ultimo file in lista). Sono definiti come campi calcolati in modo che l'installazione di l10n_it_fatturapa_out richiede solo l10n_it_fatturapa e l10n_it_fiscal_payment_term. Così facendo i campi in fattura si riferiscono a quelli definiti nei termini di pagamento (di default).
Installando opzionalmente il modulo l10n_it_fiscal_payment_mode, se nella fattura è impostata la modalità di pagamento e in quella modalità è definito anche la modalità fiscale viene utilizzata quella in fase di esportazione, altrimenti rimane quella definita nel termine di pagamento.

Reply:
Ma questa riga `invoice.fatturapa_pt_id.code` dà errore, se `l10n_it_fiscal_payment_mode` non è installato